src/modules/forms/plugins/RateLimit/RateLimitPlugin.js

```diff
@@ -19,12 +20,23 @@ const propTypes = {
     handlePluginExclude: PropTypes.func.isRequired,
 };
 
-const RateLimitPlugin = ({ className, name, handlePluginExclude, plugin, pluginName }) => {
+const RateLimitPlugin = ({
+    className,
+    name,
+    handlePluginExclude,
+    plugin,
+    pluginFromValues,
+    pluginName,
+    response,
+}) => {
     const b = block(className);
-    const optionsTransformer = config => config.map(item => ({
-        label: item,
-        value: item,
-    }));
+    const optionsTransformer = config => {
```

Comment:
Fairly I'd rather the way it was before, but that's fine

Reply:
👍